packages/web/src/scss/components/Modal/_ModalBody.scss

```diff
@@ -0,0 +1,25 @@
+@use 'sass:map';
+@use '../../tools/breakpoint';
+@use 'theme';
+
+.ModalBody {
+    flex: 1 1 auto;
+    min-height: map.get(theme.$typography, mobile, font-size) * map.get(theme.$typography, mobile, line-height);
```

Comment:
Also because we should act like we don't know the actual token value, I think. So this might now work if there were different sizes for different breakpoints.

Reply:
I can see several options here:

1. Store current font size and line height generated by `typography.generate($custom-properties: true)` (turn it on to prevent unused code elsewhere) also in a custom property, e.g. `--font-size`, `--line-height`. Then it can be consumed here so the dimension is always exact.
2. Change it to a reasonable expectation of a single-line height, e.g. `1.5em` (which is close to the current outcome of `1.5rem`).
3. Something in between: Leave it as it is. It's an edge case after all, it doesn't need to be exact. (Many other DS don't cover it at all.)

Reply:
1. sounds nice, but we can do it later :)